### PR TITLE
Docs for prtagbuilder.

### PR DESCRIPTION
**Pull request by dekiel**
Added documentation for prtagbuilder tool.

See #2693

### CODE REVIEW

File: development/tools/pkg/prtagbuilder/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# PR-tag builder
+
+## Overview
+
+**`prtagbuilder`** is a tool which will find a pull request number for a commit.
+
+## Prerequisites
+
+Tool is called on postsubmit Prow jobs. You need Prow instance to use it.
```
[the remark this answers is not in the document]
Comment (dekiel) [reply development/tools/pkg/prtagbuilder/README.md]: postsubmit is a job type in a Prow. It should not be written with dash. Other changes commited.

File: development/tools/pkg/prtagbuilder/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Installation
+
+Tool is a part of test-infra **prow-tools** image. It is installed in almost all test-infra images under `/prow-tools` path by copying from **prow-tools** image.
+
+## Usage
+
+Tool doesn't accept any flags and arguments. All data required for the tool to work, is retrived from environment variable **`JOB_SPEC`**. This environment variable is set by Prow for postsubmit jobs.`
```
[the remark this answers is not in the document]
Comment (dekiel) [reply development/tools/pkg/prtagbuilder/README.md]: postsubmit is a job type in a Prow. It should not be written with dash. Other changes commited.

File: development/tools/pkg/prtagbuilder/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Tool will fail on any error which will prevent it from finding valid PR number for a commit.
+
+Tool will not work on presubmit jobs.
```
[the remark this answers is not in the document]
Comment (dekiel) [reply development/tools/pkg/prtagbuilder/README.md]: presubmit is a job type in a Prow. It should not be written with dash. Other changes commited.

File: development/tools/pkg/prtagbuilder/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Development
+
+Changes in `prtagbuilder` source code will trigger Prow presubmit and postsubmit jobs. Jobs will run tests and build **prow-tools** image. Version of **prow-tools** image should be updated in test-infra images dockerfiles, by replacing image tag to match new version. This will trigger rebuild of images which copy from **prow-tools** image.
```
[the remark this answers is not in the document]
Comment (dekiel) [reply development/tools/pkg/prtagbuilder/README.md]: presubmit and postsubmit are a job types in a Prow. They should not be written with dash. Other changes commited.

File: development/tools/pkg/prtagbuilder/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+The tool is called on postsubmit Prow jobs. You need a Prow instance to use it.
+
+## Installation
+
+The tool is a part of the `test-infra` **prow-tools** image. It is copied from the **prow-tools** image and installed in almost all images under the `test-infra/prow/images/prow-tools` path.
+
+## Usage
+
+The tool doesn't accept any flags and arguments. All data required for the tool to work is retrieved from the  **JOB_SPEC** environment variable. This environment variable is set by Prow for postsubmit jobs.`
```
Comment (klaudiagrz) [inline development/tools/pkg/prtagbuilder/README.md:17]:
```suggestion
The tool doesn't accept any flags and arguments. All data required for the tool to work is retrieved from the  **JOB_SPEC** environment variable. This environment variable is set by Prow for postsubmit jobs.
```

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**